packages/domain-search/src/components/domains-mini-cart/style.scss

```diff
@@ -1,23 +1,30 @@
 @import '@wordpress/base-styles/breakpoints';
 @import '@wordpress/base-styles/mixins';
 
+.domains-mini-cart__cushion {
+	margin-top: 36px;
```

Comment:
The small detail, I would use `2.25 rem` instead of `px` here.

Reply:
@bogiii we're rendering it on purpose to avoid the screen jumping/results going behind the cart if the user adds it.